### PR TITLE
Added the susi themed nav menu and nav menu icon

### PR DESCRIPTION
**Pull request by stealthanthrax**
commit

### CODE REVIEW

**Comment by stealthanthrax** [comment]
I am still working on issue #82

File: src/style.css [hunk 1]
```diff
@@ -177,3 +177,49 @@ body{
...
+}
+.dropdown-content{
+    display: none ;
+    position: absolute ; 
+    background-color: #f9f9f9 ; 
+    min-width: 150px ;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    padding: 12px 16px ;
+    z-index: 1 ;
+    color: grey !important;
```
[the remark this answers is not in the document]
Comment (ms10398) [reply src/style.css]: @stealthanthrax Please fix the codacy issues

**Comment by ms10398** [comment]
The branch has merge conflicts with the origin please fix the conflicts too @stealthanthrax

**Comment by stealthanthrax** [comment]
@ms10398 how do I update a pr?

**Comment by stealthanthrax** [comment]
@ms10398 i have opened another pr i.e.#92.
Could you please check it out?

**Pull request closed without merging**